### DISCUSSION TITLE
Is it possible to have more than one layer in the same wms-service of orthophotos?

### DISCUSSION THREAD

**Discussion by sebros123** [INSPIRE-MIF/helpdesk#124] [Q&A]
Hello!

According to the specification "inspire_dataspecification_oi_v3.0" it looks like this:

![ortotg](https://user-images.githubusercontent.com/92439786/143828792-b564a169-ec20-4bea-9a2c-fd98039a0cfc.jpg)

Lantmäteriet has several different orthophotos in the same WMS-service, how should one think here?
Is OI.OrthoimageCoverage a prefix that can be used? Or does the spec say that there should only be one orthophoto per viewing service?

Mvh Sebastian

**Comment by sMorrone** [comment]
Dear @sebros123, 
it is possible to provide several different orthophotos in the same WMS-service. You can create a "Layer Group" named **OI.OrthoimageCoverage** and provide the orthophotos as layers of this group. There is no fixed naming rule for the layers in the group.

**Comment by sebros123** [comment]
Thank you very much for your answer, it helps us to move forward :)

**Comment by sMorrone** [comment]
@sebros123 
can we proceed with closing this issue? (Clearly you could re-open / open a new one should you have any other questions)

**Comment by sebros123** [comment]
@sMorrone 
Yes we can do that, thank you

**Comment by sebros123** [comment]
Hi again! @sMorrone  

The name of the top layer is ‘OI.OrthoimageCoverage’ but the ETF validator still says we have bad layer names for the layers inside the group. 

You wrote earlier that "There is no fixed naming rule for the layers in the group". Is it the case that the layers must have the prefix OI.? 

Or can ju give me a hint how the layergroup must be configured?

**Comment by sMorrone** [comment]
@sebros123 
could you provide the WMS service endpoint (so that we can have a look and in case open an issue in the INSPIRE-validator helpdesk)?

**Comment by sMorrone** [comment]
@jescriu 
could you provide your OI expert support to this discussion? Thank you

**Comment by sebros123** [comment]
@sMorrone @jescriu

Here is the GetCapabilities-file from the production ortofoto WMS-service. 
[Orthofoto_Inspire_WMS_TEST.txt](https://github.com/INSPIRE-MIF/helpdesk/files/7984233/Orthofoto_Inspire_WMS_TEST.txt)

I can ask if our developer can join the diskussion

**Comment by fabiovinci** [comment]
Dear @sebros123,

please have a look at [this comment](https://github.com/INSPIRE-MIF/helpdesk-validator/issues/39#issuecomment-974206886) which explains the status of this issue.

I tried to validate your WMS in the [staging instance](http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/home/index.html), using the local copy of the GetCapabilities document, and there is no error for the "at39-getcapabilities-layer-name" test.

So that means your implementation is right.

Please do a test using the online version of the GetCapabilities document and let us know if everything is ok.

**Comment by sebros123** [comment]
@fabiovinci  Thank you,

I'll do it and get back to you. 

// Sebastian

**Comment by jescriu** [comment]
Dear @sMorrone,

Although this issue was originated with a view service related to the the Orthoimagery theme, I think that the overlying problem is related to implementation of view services in general.

Thanks @fabiovinci for pointing to [Issue #39](https://github.com/INSPIRE-MIF/helpdesk-validator/issues/39) in the helpdesk of the INSPIRE Reference Validator, which is in the pipeline for addressing the problem.
